tokenizers/src/pre_tokenizers/metaspace.rs

```diff
@@ -282,14 +283,9 @@ mod tests {
                 .map(|(s, o, _)| (s, o))
                 .collect::<Vec<_>>(),
             vec![
-                ("▁Hey", (0, 6)),
-                ("▁my", (6, 11)),
-                ("▁friend", (11, 20)),
-                ("▁", (20, 23)),
+                ("▁Hey▁my▁friend▁", (0, 23)),
                 ("<s>", (23, 26)),
-                ("how", (26, 29)),
-                ("▁are", (29, 35)),
-                ("▁you", (35, 41))
+                ("how▁are▁you", (26, 41))
```

Comment:
this means meta space does not necessarily split on whitspace. Seems to work for BPE / Llama idk